### DISCUSSION TITLE
Sorting sidebar navigation alphabetically not working

### DISCUSSION THREAD

**Discussion by AndyRaphael** [imfing/hextra#312] [Q&A]
Hello

I am not sure if this is a bug or a setting on my end, but items in the sidebar navigation are not sorted alphabetically, but rather by date (newest on top). This is happening in subfolders and none of the page use weights. How could I sort them alphabetically?

Thanks!

**Comment by AndyRaphael** [comment]
One clarification: When removing the default "date" parameter from front matter, then it sorts alphabetically.

**Reply by imfing** [reply]
Currently, it's sorted by the weight of the entries. Maybe we should expose an option to allow different sort order.
